packages/zarf-agent/zarf.yaml

```
@@ -30,7 +30,7 @@ components:
     actions:
       onCreate:
         before:
-          - cmd: "make init-package-local-agent AGENT_IMAGE_TAG=\"###ZARF_PKG_TMPL_AGENT_IMAGE_TAG###\" ARCH=\"###ZARF_PKG_ARCH###\""
+          - cmd: "test \"###ZARF_PKG_TMPL_AGENT_IMAGE_TAG###\" != \"local\" || make build-local-agent-image AGENT_IMAGE_TAG=\"###ZARF_PKG_TMPL_AGENT_IMAGE_TAG###\" ARCH=\"###ZARF_PKG_ARCH###\""
```

Hehehe - `The token '||' is not a valid statement separator in this version.`

```suggestion
          - cmd: "test \"###ZARF_PKG_TMPL_AGENT_IMAGE_TAG###\" != \"local\" || make build-local-agent-image AGENT_IMAGE_TAG=\"###ZARF_PKG_TMPL_AGENT_IMAGE_TAG###\" ARCH=\"###ZARF_PKG_ARCH###\""
            shell:
              windows: cmd
```

Setting a preferred windows shell should fix it.

---

(although `test` may not exist on GH runners since it is not default in Windows)

---

Following up on this `mingw` is available on the runners so it should work - https://github.com/actions/runner-images/blob/main/images/win/Windows2019-Readme.md#tools